web/controllers/github_api/http_client.ex

```diff
@@ -129,4 +133,22 @@ defmodule Dwylbot.GithubAPI.HTTPClient do
     end)
   end
 
+  def get_data(token, payload, "issue_from_status") do
+    pull_request_payload = %{
+      "repository" => %{
+        "full_name" => payload["repository"]["full_name"]
+      }
+    }
+    pull_requests = get_data(token, pull_request_payload, "list_pull_requests")
+    branch_name = payload["branches"]
+    |> List.first()
```

Comment:
After looking into this a little, I think that adding `()` is becoming the done thing according to [this styleguide](https://github.com/christopheradams/elixir_style_guide/issues/144#issuecomment-309256629).

I think I'll have to take back my original complaints about parens on 0-arity piped functions, these seem like the way to go